pkg/repository/prisma/dbsqlc/sqlc.yaml

```diff
@@ -25,7 +25,7 @@ sql:
       - queue.sql
       - lease.sql
     schema:
-      - schema.sql
+      - ../../../../sql/schema/schema.sql
```

Comment:
what is happening here now... why did this ref change?

Reply:
iirc we were processing the schema and moving it to this location, now we are just using the schema and skipping the prisma piece